python/tests/unit/memory/test_qdrant_memory_store.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from pytest import mark, raises
+
+from semantic_kernel.memory import QdrantMemoryStore
```

Comment:
```suggestion
from semantic_kernel.memory.qdrant import QdrantMemoryStore
```

Reply:
Update unit test and moved to test/integration/connectors 
Addressed in commit: https://github.com/microsoft/semantic-kernel/pull/919/commits/643457c6879902dcb849378ebb171a5d65d16b47